### DISCUSSION TITLE
Can keras 3 limit the device memory?

### DISCUSSION THREAD

**Discussion by jeffDebug** [keras-team/keras#19935] [Q&A] [answered]
This may not be a reasonable question. But I thought it wouldn't hurt to give a try.

Is there any API calls, in keras 3, that is equivalent to the following in tensorflow?
`tensorflow.config.set_logical_device_configuration(gpu, [tensorflow.config.LogicalDeviceConfiguration(memory_limit=maxMB)])`

 I'm looking for the configuration API that works for all backends (tensorflow and torch). Thanks!

**Comment by santi-mir** [comment, marked as answer]
I don't know of one. You can write if/else that checks the value `keras.backend()` and sets whatever you need.
